ui/components-react/src/components-react/filter-builder/FilterBuilder.tsx

```diff
@@ -150,12 +154,13 @@ export function PropertyFilterBuilderRenderer(
         <ActiveRuleGroupContext.Provider
```

Comment:
I think this `ActiveRuleGroupContext` is not user anywhere. Could you remove it and all related code?